core/src/test/java/org/apache/calcite/test/SqlToRelTestBase.java

```diff
@@ -169,6 +170,13 @@ public static void assertValid(RelNode rel) {
      */
     RelRoot convertSqlToRel(String sql);
 
+    /**
+     * Converts a expression string to  {@link RexNode}.
+     *
+     * @param expr expression
```

Comment:
a => an

Reply:
Thank for you reminding me

Reply:
param comments should start with uppercase character. `expression` => `The expression`.